app/routes/docs.client.samples.md

```diff
@@ -164,3 +166,62 @@ for message in consumer.consume(end[0].offset - start[0].offset, timeout=1):
             continue
     print(message.value())
 ```
+
+## Working With JSON Schema
```

Comment:
Please rename this section and the previous one to make it clear that one section is dealing with decoding XML data and the other is dealing with JSON data.